recipe/meta.yaml

```diff
@@ -25,6 +25,8 @@ requirements:
     - itsdangerous 0.24 py27_1  # [py27]
     - itsdangerous 0.24 py35_1  # [py35]
     - itsdangerous 0.24 py36_1  # [py36]
+    - werkzeug 0.12.2 py35_0  # [py35]
```

Comment:
Why does this need to be pinned to py35 specifically?

Reply:
To be honest I do not remember the reason - they were problematic for py<36. The multiple commits are due to the squashed merge earlier in the history. I should probably drop my fork and re-fork to my repo. Note that the only changes now are in other places. All the other commits are already applied earlier. 
I will squash merge and re-fork to avoid the same mess next time.

Reply:
I have found the reason. Both packages were recently packaged as noarch and these do not work. I thought it is only for build-time but it turned out to be a problem at runtime as well. Fixing in next commit.

Reply:
Can you raise an issue with either the recipe or conda-forge about the noarch not working? I'd imagine that this is unexpected/bad behavior (or at least I hope it is!).

Reply:
I did with both recipes. I do not know where to submit issues for conda-forge itself. I will submit something against feedstocks repo.